### PR TITLE
More appropriate template for notifications about new NetBeans version

### PR DESCRIPTION
**Pull request by jkovalsky**
Matthias/Neil, please review and approve my changes in the template for notifications which we will be sending to plugin developers before announcements about new NetBeans release candidate builds.

Thanks a lot!

### CODE REVIEW

File: pp3/module/Application/src/Application/Controller/AdminController.php
Comment (matthiasblaesing) [inline pp3/module/Application/src/Application/Controller/AdminController.php:375]: This will fail in most cases. The URL `/plugin/list` will redirect you to the front page because you are not logged in. This might confuse users reading the sentence directly before the link.

**Review by matthiasblaesing — COMMENTED** [review]
In general the updated message template is ok from my POV. Problematic is the new parameter `%4$s`. There is no basis for its replacement and thus must be provided by the human sending the message.

File: pp3/module/Application/src/Application/Controller/AdminController.php
Comment (jkovalsky) [reply pp3/module/Application/src/Application/Controller/AdminController.php:375]: You are right but I would not underestimate plugin developers.  For those already logged in the link will work and the others will quickly figure out what happened and navigate to My Plugins view.

Or are you suggesting to just provide the link to Plugin Portal homepage?

Comment (matthiasblaesing) [reply pp3/module/Application/src/Application/Controller/AdminController.php:375]: I can live with it. My perspective was, that at most time I would hit the redirect case, not the "direct to plugin list" case. But I have no strong feelings here.

Comment (jkovalsky) [reply pp3/module/Application/src/Application/Controller/AdminController.php:375]: Thank you Matthias. In such case I would like to merge this so that you can redeploy the Plugin Portal sources ASAP in order to inform all plugin owners about NB17 RC1. Please approve if you agree.

**Comment by jkovalsky** [comment]
@matthiasblaesing / @neilcsmith-net - can I get at least 1 approval please?

**Comment by neilcsmith-net** [comment]
Looks fine to me, but don't have enough context of usage in the portal to want to approve.  Also looks like you addressed @matthiasblaesing comment, so if we need this in urgently, I'd proceed.

**Review by matthiasblaesing — APPROVED** [review]
Sure you can :+1: 

Will merge and then immediately deploy.

**Comment by matthiasblaesing** [comment]
Deployment is done and I checked, that he new text is shown.

**Pull request merged**
